Test/FMI3/fmi3_import_options_test.cpp

```diff
@@ -141,31 +122,21 @@ int main(int argc, char *argv[])
     callbacks.free = free;
     callbacks.logger = jm_default_logger;
     callbacks.log_level = jm_log_level_debug;
-    callbacks.context = NULL;
+    callbacks.context = nullptr;
 
 #ifdef FMILIB_GENERATE_BUILD_STAMP
     printf("Library build stamp:\n%s\n", fmilib_get_build_stamp());
 #endif
 
-    tmpPath = fmi_import_mk_temp_dir(&callbacks, FMU_UNPACK_DIR, NULL);
-    if (!tmpPath) {
-        printf("Failed to create temporary directory in: " FMU_UNPACK_DIR "\n");
-        do_exit(CTEST_RETURN_FAIL);
-    }
+    tmpPath = fmi_import_mk_temp_dir(&callbacks, FMU_UNPACK_DIR, nullptr);
+    REQUIRE(tmpPath != nullptr);
 
     context = fmi_import_allocate_context(&callbacks);
     version = fmi_import_get_fmi_version(context, FMU3_ME_PATH, tmpPath);
-    if (version != fmi_version_3_0_enu) {
-        printf("Only version 3.0 is supported by this code\n");
-        do_exit(CTEST_RETURN_FAIL);
-    }
-
-    fmu = fmi3_import_parse_xml(context, tmpPath, NULL);
+    REQUIRE(version == fmi_version_3_0_enu);
 
-    if (!fmu) {
-        printf("Error parsing XML. Exiting.\n");
-        do_exit(CTEST_RETURN_FAIL);
-    }    
+    fmu = fmi3_import_parse_xml(context, tmpPath, nullptr);
+    REQUIRE(fmu != nullptr);
```

Comment:
Great, looks like this simplify the tests a bit.  How do the error messages looks from this? Do REQUIRE generate good assert failed messages?

Reply:
okey, good then.